src/client/sandbox/node/window.js

```diff
@@ -128,11 +128,17 @@ export default class WindowSandbox extends SandboxBase {
         }
 
         if (window.Worker) {
-            window.Worker           = scriptURL => {
+            window.Worker           = function (scriptURL, options) {
+                if (this instanceof window.Worker === false)
```

Comment:
After moving check to a separate method we with @churkin decide that comment is unnecessary.
So @LavrovArtem review this